goosebit/settings/schema.py

```diff
@@ -61,9 +67,23 @@ def settings_customise_sources(
         dotenv_settings: PydanticBaseSettingsSource,
         file_secret_settings: PydanticBaseSettingsSource,
     ) -> tuple[PydanticBaseSettingsSource, ...]:
-        return (
+        settings_sources = [
             init_settings,
-            YamlConfigSettingsSource(settings_cls, BASE_DIR.joinpath("settings.yaml")),
-            env_settings,
-            file_secret_settings,
+            YamlConfigSettingsSource(settings_cls, GOOSEBIT_ROOT_DIR.joinpath("goosebit.yaml")),
+            YamlConfigSettingsSource(settings_cls, CURRENT_DIR.joinpath("goosebit.yaml")),
+        ]
+        if ETC_SETTINGS_DIR.exists():
+            settings_sources.append(
+                YamlConfigSettingsSource(settings_cls, ETC_SETTINGS_DIR.joinpath("goosebit.yaml")),
+            )
+        if os.getenv("GOOSEBIT_SETTINGS"):
```

Comment:
```suggestion
        if os.getenv("GOOSEBIT_SETTINGS") is not None:
```

Anyway, IMHO the env var should have precedence.

Reply:
> Anyway, IMHO the env var should have precedence.

My understanding was that it does.  Seems like these are loaded first to last, and later values can overwrite earlier ones.